static/src/javascripts/projects/common/modules/identity/manage-account.js

```diff
@@ -37,17 +39,54 @@ const submitPartialFormStatus = (
     });
 };
 
+const getNewsletterHtmlPreferenceFromElement = (
+    originalEl: HTMLElement
+): Promise<string> =>
+    fastdom.read(() => {
+        const closestFormEl: ?Element = originalEl.closest('form');
+
+        if (!closestFormEl) throw Error(ERR_HTML_PREF_NOT_FOUND);
```

Comment:
`throw new Error()` ?

Reply:
gosh yes, eslint complains about `new` so much i forgot there's legit uses for it

Reply:
You are right - we should propably discuss this again because we have many of them in our codebase and ignore them every time ... @SiAdcock , @sndrs, @GHaberis what do you think?